spring-cloud-gcp-pubsub-stream-binder/src/test/java/org/springframework/cloud/gcp/stream/binder/pubsub/PubSubEmulator.java

```diff
@@ -137,6 +126,21 @@ protected void after() {
 		}
 	}
 
+	private int validateEmulator() {
```

Comment:
findAndDestroyEmulator? Or something less clunky but still descriptive...
I like the refactoring into a separate method, but it does more than validate.